_maps/RandomZLevels/research.dmm

```diff
@@ -2729,7 +2729,7 @@
 "ij" = (
 /mob/living/simple_animal/bot/secbot/beepsky{
 	desc = "A beefy variant of the standard securitron model.";
-	emagged = 1;
```

Comment:
Emagged wasn't a boolean value for /mob/living/simple_animal/bot, just an FYI.

Reply:
oh, i shouldn't have removed this